control_software/scripts/hera_snap_redis_monitor.py

```diff
@@ -367,7 +372,21 @@ def print_ant_log_messages(corr):
                     redis_vals["fft_of"] = fft_of[key]
                 except KeyError:
                     pass
+
                 redis_vals['timestamp'] = datetime.datetime.now().isoformat()
+
+                for key in redis_vals:
+                    if isinstance(redis_vals[key], bool):
+                        # bools are compared using lambda x: x == "True" later
+                        redis_vals[key] = str(redis_vals[key])
+                    elif isinstance(redis_vals[key], list):
+                        # values that are appearing as lists as loaded
+                        # with json.loads in corr_cm
+                        redis_vals[key] = json.dumps(redis_vals[key])
+                    elif redis_vals[key] is None:
+                        # newer redis-py does not accept Nonetype, wrap in json.dumps
+                        redis_vals[key] = json.dumps(redis_vals[key])
+
```

Comment:
Is it possible to blow through this list without hitting one of these clauses? Like do we need an `else` branch?

Reply:
hm. these were all the cases I found that were breaking redis, the `else` is just "this type works with redis" could add an explicit `pass` maybe?

Reply:
Nope, that's fine. Maybe we can just add a comment at the top of this `if` statement that we're doing explicit type conversion.

Reply:
Made a comment inside the for loop about this.